resources/todomvc/vanilla-examples/javascript-es6/dist/view.js

```diff
@@ -66,11 +62,19 @@ class View {
         this.$toggleAll = qs(".toggle-all-label");
         this.$toggleAllInput = qs(".toggle-all");
         this.$newTodo = qs(".new-todo");
+        function replaceContentWithHtml(element, html) {
+            // This has been adopted from
+            // https://github.com/unpoly/unpoly/blob/78b1dc0d5729d083e7cb9dfcb73272451c0a48a8/src/unpoly/element.js#L755-L773
+            const range = document.createRange();
+            range.setStart(document.body, 0);
+            const fragment = range.createContextualFragment(html.trim());
+            element.replaceChildren(...fragment.childNodes);
+        }
```

Comment:
I just copied pasted this from `unpoly` directly.

Reply:
If we're copying code like this, we need to include the licensing.
Could you add https://github.com/unpoly/unpoly/blob/master/LICENSE to somewhere?

Reply:
I was thinking that just the pointer as a comment would be enough for this, because this isn't a "substantial portion of the software". What do you think?

Reply:
Hm... I'm not a lawyer so won't able to tell you either way.